### PR TITLE
add ci

### PR DESCRIPTION
**Pull request by chrbeckm**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          # add to path for following steps
+          echo "$HOME/texlive/$TEXLIVE_VERSION/bin/x86_64-linux" >> $GITHUB_PATH
+          export PATH="$HOME/texlive/$TEXLIVE_VERSION/bin/x86_64-linux:$PATH"
+          tlmgr install --with-doc tex-overview
+
+          curl -LO https://www.wfonts.com/download/data/2014/06/05/comic-sans-ms/comic-sans-ms.zip
+          unzip comic-sans-ms.zip -d $HOME/.local/share/fonts
+          rm comic-sans-ms.zip
+          fc-cache
+          fc-list 'Comic Sans MS' | grep Comic
```
Comment (maxnoe) [inline .github/workflows/ci.yml:49]: comic sans is not needed for the template repo, I hope :D

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# https://github.com/orgs/community/discussions/22183#discussioncomment-6321583
+
+name: first-time-setup
```
Comment (maxnoe) [inline .github/workflows/deploy.yml:3]: What are we deploying here?

Comment (maxnoe) [reply .github/workflows/deploy.yml:3]: Maybe call this "remove_template_files.yml" or something?

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      - name: Checkout code
+        uses: actions/checkout@v3
+        # ADDED THIS CHANGE
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # remove files not needed for user instance of template
+      - name: Remove unneeded files
+        run: |
+          mv README.md notes.md
```
Comment (maxnoe) [inline .github/workflows/deploy.yml:40]: Do we want to add a dummy readme instead of removing it?

**Review by aknierim — APPROVED** [review]

**Pull request merged**